h/models/group.py

```diff
@@ -100,6 +100,13 @@ def slug(self):
     def is_public(self):
         return self.readable_by == ReadableBy.world
 
+    @property
+    def logo(self):
+        """Returns logo icon if one exists for this groups authority else
+        returns default logo."""
+        # todo: find an appropriate default logo
+        return {}.get(self.authority, 'logo')
```

Comment:
Should providing a default logo be the responsibility of the model?

Reply:
I think it's nice to keep it in one location. I anticipate activity pages won't be the only location we use the logo in the future. This is also a question for @dawariley: What should we display if a logo for that authority doesn't exist? (aka: If we don't want to display a logo then we don't need to provide a default logo and it can be left as None.)

Reply:
Spoke with Dawa a moment ago- She says don't display the logo if we don't have one so I'll update the code to reflect that.

Reply:
Thanks @hmstepanek. Yes this was the consideration for activity pages. I'm assuming most publishers/partners will supply us with a logo. If the logo is missing from the sidebar I think the same should apply no placeholder - although it will look 'plain' it doesn't break the design to not have a logo. If it becomes an issue we could tackle it at a future stage. It might become necessary when we have self serve for groups because an organization's logo might be deleted/wrong format/unable to display etc. Rather than showing a broken image link we could then decide to show a placeholder or perhaps just hide the logo. I'm curious to know what happens now if the SVG can't be found or displayed - how the code handles it?